app/controllers/movie_controller.rb

```diff
@@ -7,6 +9,11 @@ def search
     @movies = MovieFacade.search(params[:query])
   end
 
+  def show
+    @user = current_user
+    @movie = MovieFacade.movie_details(params[:id].to_i)
```

Comment:
Absolutely beautiful! Look at that single line that is going to do so much work for calling a movies details!